packages/noya-renderer/src/components/EditablePath.tsx

```diff
@@ -85,13 +126,16 @@ export default function EditablePath({
 
   if (!Layers.isPointsLayer(layer)) return null;
 
-  const points = layer.points.map(
-    (point) => Primitives.parseCurvePoint(point, layer.frame).point,
+  const parsedCurvePoints = layer.points.map((point) =>
+    Primitives.parseCurvePoint(point, layer.frame),
```

Comment:
Remember to merge/rebase on main, since we renamed this to `decodeCurvePoint`, and also changed the `POINT_SIZE` => `POINT_RADIUS` above

Reply:
Done.